src/main/java/org/intellij/sdk/codesync/PopulateBuffer.java

```diff
@@ -158,7 +158,7 @@ public static Map<String, String> detectBranchChange() {
         try {
             configFile = new ConfigFile(CONFIG_PATH);
         } catch (InvalidConfigFileError error) {
-            CodeSyncLogger.logEvent(String.format(
+            CodeSyncLogger.critical(String.format(
                     "[INTELLIJ_PLUGIN][POPULATE_BUFFER] Config file error, %s.\n", error.getMessage()
```

Comment:
Question, why do we add `[INTELLIJ_PLUGIN]` in the logs when we are already sending INTELLIJ as source?

Reply:
Yes, I think we should remove this.
I will remove it in the next PR.

Reply:
Cool, thanks